stdlib/fs.gr

```diff
@@ -0,0 +1,1045 @@
+/**
+ * High-level file system interactions. Utilizes WASI Preview 1 for underlying API
+ * 
+ * @example from "fs" include Fs
+ * 
```

Comment:
I think it would be beneficial to add some quick usage examples to the top of the library for what will probably be the most commonly used functions, similary to what we do in [The has module](https://grain-lang.org/docs/stdlib/hash)
```suggestion
 * @example Fs.Utf8.readFile(Path.fromString("baz.txt"))
 * @example Fs.Utf8.writeFile(Path.fromString(("baz.txt"), "Hello Grain File's")
```